rclcpp/include/rclcpp/timer.hpp

```diff
@@ -95,11 +97,9 @@ using TimerCallbackType = std::function<void (TimerBase &)>;
 /// Generic timer templated on the clock type. Periodically executes a user-specified callback.
```

Comment:
doc needs update

Reply:
Updated in 04621963e960f136f058d3430ed485ade457e5c6.